src/ApiTestCase.php

```diff
@@ -112,6 +108,14 @@ public function tearDown()
         parent::tearDown();
     }
 
+    /**
+     * return ProcessorInterface[]
+     */
+    protected function getFixtureProcessors()
```

Comment:
I cannot find a usage here. Is it really necessary to has this method in a main file?